### PR TITLE
Slim down README.md and make tutorials page

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

This moves the tutorials down underneath their own page and directory in the docs directory to emulate what we did with Buildah.  In addition I slimmed down README.md to make it a quick read for someone wanting to know about the project.  The commands took up a lot of space so I've moved them to their own page.  We did similar changes with Buildah. The gist is to provide them with high level information on the README,md with the real meat to be found with links from it.

More tutorials are in the works, thus I left the verbiage as plural there.  I plan to create another example based on the examples in the podman run man page.

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat You have whitespace errors.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan thx, think I just nixed them.  Empty line at the end of two files.

File: README.md [hunk 1]
```diff
@@ -30,45 +30,11 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
-| [podman-top(1)](/docs/podman-top.1.md)                   | Display the running processes of a container              |[![...](/docs/play.png)](https://asciinema.org/a/5WCCi1LXwSuRbvaO9cBUYf3fk)|
-| [podman-umount(1)](/docs/podman-umount.1.md)             | Unmount a working container's root filesystem                             |[![...](/docs/play.png)](https://asciinema.org/a/MZPTWD5CVs3dMREkBxQBY9C5z)|
-| [podman-unpause(1)](/docs/podman-unpause.1.md)           | Unpause one or more running containers                                    |[![...](/docs/play.png)](https://asciinema.org/a/141292)|
-| [podman-version(1)](/docs/podman-version.1.md)           | Display the version information                                           |[![...](/docs/play.png)](https://asciinema.org/a/mfrn61pjZT9Fc8L4NbfdSqfgu)|
-| [podman-wait(1)](/docs/podman-wait.1.md)                 | Wait on one or more containers to stop and print their exit codes  |[![...](/docs/play.png)](https://asciinema.org/a/QNPGKdjWuPgI96GcfkycQtah0)|
+**[Podman Commands](commands.md)**
```
Comment (baude) [inline README.md]: I would like to see this section cleaned up if possible.  As of now, it looks like an orphaned list of links.

File: README.md [hunk 2]
```diff
@@ -30,45 +30,11 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+
+**[Installation notes](install.md)**
+
+**[Tutorials](docs/tutorials/tutorials.md)**
```
Comment (baude) [inline README.md]: Can we have a small description of the tutorial around this?  And perhaps in line with the above comment, somehow reformat/reorganize to make it pop a little more

File: README.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply README.md]: Think I've done so, open to further suggestions.  Will repost shortly.

File: README.md (hunk 2 above)
Comment (TomSweeneyRedHat) [reply README.md]: Ditto previous comment.

**Comment by TomSweeneyRedHat** [comment]
Going to close/reopen to rekick the tests.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
@rhatdan can this be merged?  Red Hat CI is still failing, but absolutely no logs/errors to show what's up.

**Comment by rhatdan** [comment]
Merging since this is a docs change only and cento and travis are fine.  Not sure what is broken in the infrastructure.

**Pull request merged**